plugins/toolkit/jetbrains-core/src-233/software/aws/toolkits/jetbrains/core/explorer/ActionToolbarLayoutUtil.kt

```diff
@@ -0,0 +1,11 @@
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.aws.toolkits.jetbrains.core.explorer
+
+import com.intellij.openapi.actionSystem.ActionToolbar
+
+object ActionToolbarLayoutUtil {
```

Comment:
We probably dont need this object, it can be a standalone function

Reply:
got it